sys/src/9/port/devramfs.c

```diff
@@ -145,6 +194,10 @@ ramopen(Chan *c, int omode)
 		nexterror();
 	}
 	Chan* ret = devopen(c, omode, nil, 0, ramgen);
+	struct RamFile* file = (struct RamFile*)c->qid.path;
```

Comment:
To me this looks like a panic. In what situation is this not a serious bug or memory corruption?